src/JoomlaBrowser.php

```diff
@@ -855,4 +856,16 @@ public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "/
 		$I->assertEquals($expectedTabs, $actualArrayOfTabs, "Tab Labels do not match on edit view of" . $url);
 		$I->debug('Verify the Tabs');
 	}
+
+    /**
+     * Hide the statistics info message
+     *
+     * @note: doAdminLogin() before
+     */
+    public function disablestatistics()
```

Comment:
Nice feature! good job.

we need to fix coding standards in all JoomlaBrowser but lets take the chance here before it get merge. Please update the current name `disablestatistics` to `disableStatistics `